### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by ttahm3d**
- Created Header
- Created Footer
- Added content onto homepag

### CODE REVIEW

File: src/views/Homepage/Homepage.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    text-align: center;
+
+    h1 {
+      font-size: 3rem;
+      color: hsla(var(--accent-light), 1);
+    }
+
+    h2 {
+      font-size: 1.4rem;
+      padding-top: 0.75rem;
```
Comment (kjais1720) [inline src/views/Homepage/Homepage.scss:23]: you could use the variables from the component library for font-sizes

File: src/views/Homepage/Homepage.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    .hero-buttons {
+      margin-top: 2rem;
+    }
+  }
+}
+
+.grid-eq-auto {
+  grid-template-columns: repeat(auto-fit, minmax(15rem, 1fr));
+}
```
Comment (kjais1720) [inline src/views/Homepage/Homepage.scss:35]: Why not use the grid utilities from the component library

**Review by kjais1720 — APPROVED** [review]
Looks good just a few small suggestions, also please include a screenshot or a gif of the page in the PR as well

File: src/views/Homepage/Homepage.scss (hunk 2 above)
Comment (ttahm3d) [reply src/views/Homepage/Homepage.scss:35]: the minimum width is too small for small screens

File: src/views/Homepage/Homepage.scss (hunk 1 above)
Comment (ttahm3d) [reply src/views/Homepage/Homepage.scss:23]: I specifically wanted to decrease the font size as it was too large from componet library

**Pull request merged**